.bin/srclib-javascript

```diff
@@ -181,9 +181,7 @@ parser.command("depresolve")
   })
   .help("resolves dependencies of a CommonJS package");
 
-var hardcodedRepoURLs = {
```

Comment:
Will this make it so `srclib-javascript` will be unable to resolve the node standard lib to the repository URL? We eventually want to move away from repository URLs, but in the meantime, they are the standard way to identify repositories and resolve dependencies.